### DISCUSSION TITLE
Issue with adding selected properties

### DISCUSSION THREAD

**Discussion by kjdahldev** [microsoft-search/pnp-modern-search#4008] [General]
**Version used**
Ex: 4.0

**Describe the bug**
When manually adding properties (since the list returns too many) to the field Selected properties, the added property gets combined with all already added properties. This leads to the property not working. If I delete all properties and add just the one, it will work but not the second one added. Last time I used this web part it was fine so I am not sure if its environment related or what. 

When I inspect CanvasContent1 on the page item, this is the result:
<img width="1555" alt="selectedProps" src="https://github.com/user-attachments/assets/25e54598-437c-47bc-88a0-7e3c119b13ce">

Using a script I can force the properties to be added correctly as unique strings in the array but its a bit cumbersome ;)
<img width="511" alt="fix" src="https://github.com/user-attachments/assets/be6be8b9-e42c-408a-8616-45bffe9de248">

**To Reproduce**
Manual add new properties at the end of field Selected properties using format ,Prop1,Prop2

**Expected behavior**
Properties added correctly.

**Screenshots**
See above.

**Desktop (please complete the following information):**
 - Chrome

**Additional context**
-

**Comment by kasperbolarsen** [comment]
Are you on the most recent released version?
are you aware of the export/import option? This IS a workaround, but it is a much easier way to add properties than using a script
![image](https://github.com/user-attachments/assets/9098b179-47e8-4d8b-a2cd-d499e450c97a)

**Reply by kjdahldev** [reply]
No, we are on 4.0.0 so import/export is missing. So hopefully updating will sort it then. Thanks for your quick reply.
